Makefile

```diff
@@ -0,0 +1,39 @@
+APPS = asset-manager content-store govuk-content-schemas multipage-frontend \
+	publishing-api router router-api rummager specialist-frontend \
+	specialist-publisher static travel-advice-publisher
+
+all: test
```

Comment:
does the `test` here mean that it depends on the test task? (which in turn depends on all the others?